### PR TITLE
add basic ui

### LINKED ISSUE #1: Basic UI for app
- add header
- add button in center
- add a display quote box with random text below it
- add a copy button in the box ON hover
- add styles as per your ideas
- make it mobile responsive

### PR DESCRIPTION
**Pull request by nehakulkarni11**
Branch for issue #1

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+*{
+    padding: 0;
+    margin: 0;
+    background-image: linear-gradient(to bottom right, black, rgb(42, 0, 81)); 
+}
+#header{
```
Comment (mohitdhatrak) [inline styles.css]: use classes for css, id for JS

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    text-align: center;
+    background-color: rgb(70, 5, 131);
+    padding: 50px;
+    color: aliceblue;
+    font-weight: bolder;
+    font-size: 40px;
```
Comment (mohitdhatrak) [inline styles.css:30]: - use units rem, or em instead of px - read about this
- use hex codes for color instead of names

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    font-size: 20px;
+    padding: 10px 20px;
+    border-radius: 4px;
+    background-color: aliceblue;
+    color: black;
+    border: 2px solid #371F76;
+    background-image: none;
+}
+.quote-container{
+    background-image: none;
```
Comment (mohitdhatrak) [inline styles.css:144]: why is background-image: none; used in all classes?
don't repeat CSS, use inheritance if possible - read

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+#generate-btn:hover {
+    background-color: #371F76;
+    transition-duration: 0.4s;
+    color: aliceblue;
+    cursor: pointer;
+}
+#footer{
+    margin-top: 350px;
+    text-align: center;
+    background-color: rgb(70, 5, 131);
```
Comment (mohitdhatrak) [inline styles.css]: be Consistent, if using hex code for background-color, then use it everywhere, don't use rgb

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    padding: 20px;
+    color: aliceblue;
+    font-weight: bolder;
+    font-size: 20px;
+    background-image: none;
+}
+.fa {
+        padding: 20px;
+        font-size: 30px;
+        width: 30px;
```
Comment (mohitdhatrak) [inline styles.css]: same unit - px to rem or em, update all places -- read about this

File: index.html [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Random Quote Generator</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section id="gradient"></section>
+    <section id="header">Random Quote Generator</section>
```
Comment (mohitdhatrak) [inline index.html]: why is this a section? read how to use section
use h1 or similar tags for header

File: index.html [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    <div id="quote"></div>
+    <button id="generate-btn" type="button">Generate</button>
+    </div>
+
+    <section id="footer">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<a href="https://www.linkedin.com/in/nehakulkarni11/" class="fa fa-linkedin"></a>
+<a href="https://www.instagram.com/nehakulkarni._/" class="fa fa-instagram"></a>
+<a href="https://github.com/nehakulkarni11" class="fa fa-github"></a>
+<section id="socials">Let's Connect!</section></section>
```
Comment (mohitdhatrak) [inline index.html]: why is this a section?
there is also a footer tag in HTML - read

File: index.html [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    <section id="gradient"></section>
+    <section id="header">Random Quote Generator</section>
+    
+    <div class="quote-container">
+    <div id="quote"></div>
+    <button id="generate-btn" type="button">Generate</button>
+    </div>
+
+    <section id="footer">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (mohitdhatrak) [inline index.html]: change the position of this link tag!
Should be in head tag above?

**Review by mohitdhatrak — CHANGES_REQUESTED** [review]
I have commented suggestions and changes in code, check it and update see above

File: styles.css (hunk 3 above)
Comment (nehakulkarni11) [reply styles.css:144]: i did not want to override the same property everywhere, will use inheritance.

File: styles.css (hunk 4 above)
Comment (nehakulkarni11) [reply styles.css]: i chose that manually

File: index.html [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+<a href="https://www.linkedin.com/in/nehakulkarni11/" class="fa fa-linkedin"></a>
+<a href="https://www.instagram.com/nehakulkarni._/" class="fa fa-instagram"></a>
+<a href="https://github.com/nehakulkarni11" class="fa fa-github"></a>
```
Comment (mohitdhatrak) [inline index.html]: add attribute target blank
read about what it does

**Review by mohitdhatrak — APPROVED** [review]
good to merge

**Pull request merged**